modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -3023,6 +3024,7 @@ public CodegenParameter fromParameter(Parameter parameter, Set<String> imports)
             String parameterDataType = this.getParameterDataType(parameter, parameterSchema);
             if (parameterDataType != null) {
                 codegenParameter.dataType = parameterDataType;
+                imports.add(parameterDataType);
```

Comment:
Should the import below (line 3059) already do the job?

Reply:
I've changed my solution for using line 3059. It looks better.